src/Compilers/VisualBasic/Test/Semantic/Semantics/GetExtendedSemanticInfoTests.vb

```diff
@@ -6221,25 +6221,15 @@ End Class
             Dim current = DirectCast(compilation.GetSpecialType(System_Object).ContainingAssembly.GetSpecialTypeMember(SpecialMember.System_Collections_IEnumerator__Current), PropertySymbol)
             Dim dispose = DirectCast(compilation.GetSpecialType(System_Object).ContainingAssembly.GetSpecialTypeMember(SpecialMember.System_IDisposable__Dispose), MethodSymbol)
 
-            For Each useInterface In {True, False}
```

Comment:
each path here ended up invokign the exact same code (Since the type parameter passed in in the second place had no effect whatsoever).